mmagic/models/editors/animatediff/animatediff_utils.py

```diff
@@ -0,0 +1,1106 @@
+# Copyright 2023 The HuggingFace Inc. team.
```

Comment:
These comments should be removed. Make it just like your resnet_3d.py and unet_block.py

Reply:
I've removed this.